docs/main_text.md

```diff
@@ -60,31 +60,148 @@ If you would prefer to report issues via e-mail, you can also email [ccdl@alexsl
+If you would prefer to report issues via e-mail, you can also email [ccdl@alexslemonade.org](mailto:ccdl@alexslemonade.org).
+
+#### Salmon 
+
+Salmon is an alignment-free method for estimating transcript abundances from RNA-seq data ([Patro, et al. _Nature Methods_. 2017.](http://dx.doi.org/10.1038/nmeth.4197)). 
+We use it in [quasi-mapping mode](http://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-quasi-index-and-fmd-index-based-modes), which is significantly faster than alignment-based approaches and requires us to build a Salmon transcriptome index. 
+
+##### Transcriptome index
+
+We build a custom reference transcriptome (using [RSEM](https://github.com/deweylab/RSEM) `rsem-prepare-reference`) by filtering the Ensembl genomic DNA assembly to remove _pseudogenes_, which we expect could negatively impact the quantification of protein-coding genes. 
+This means we're obtaining abundance estimates for coding as well as non-coding transcripts. 
+
+Building a transcriptome index with `salmon index` requires us to specify a value for the parameter `-k` that determines the size of the k-mers used for the index.
+The length of a read determines what k-mer size is appropriate. 
+Consistent with the recommendations of the authors of Salmon, we use an index build with _k_ = 31 when quantifying samples with reads with length > 75bp.
+We use _k_ = 23 for all other read lengths.
+
+TODO: Note about how to obtain Salmon indices. Do we also make gene to transcript mapping passed to `rsem-prepare-reference` available?
+
+##### Quantification with Salmon
+
+When quantifying transcripts with `salmon quant`, we take advantage of options that allow Salmon to learn and attempt to correct for certain biases in sequencing data. 
+We include the flags `--seqBias` to correct for random hexamer priming and, if this is a **paired-end** experiment, `--gcBias` to correct for GC content when running salmon quant. 
+We set the library type parameter such that Salmon will [infer the sequencing library type automatically](http://salmon.readthedocs.io/en/latest/salmon.html#what-s-this-libtype) for the reads it is quantifying (`-l A`).
+
+#### tximport
+
+Salmon quantification is at the _transcript-level_. 
+To better integrate with the microarray data contained in refine.bio, we summarize the transcript-level information to the _gene-level_ with `tximport` ([Soneson, Love, and Robinson. _F1000 Research._ 2015.](http://dx.doi.org/10.12688/f1000research.7563.1)).
+
+Our tximport implementation generates ["lengthScaledTPM"](https://www.rdocumentation.org/packages/tximport/versions/1.0.3/topics/tximport), which are gene-level count-scale values that are generated by scaling TPM using the average transcript length across samples and to the library size. 
+Note that tximport is applied at the _experiment-level_ rather than to single samples. 
```

Comment:
Brief caveats for samples that are part of multiple experiments? Are there any? If not, ignore this comment.

Reply:
I'll add a TODO because I'm not sure.